docs/cudf/source/user_guide/missing-data.ipynb

```diff
@@ -2,46 +2,31 @@
  "cells": [
```

Comment:
<p>I removed the hardcoded table of contents below as Sphinx/myst-nb already generate one for us.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/10691/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>